src/main/java/seedu/address/model/person/Record.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_DATE_FORMAT;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+
+/**
+ * Represents a single record in the record list of a Person object.
+ */
+public class Record {
+    public static final DateTimeFormatter DATE_FORMAT = DateTimeFormatter.ofPattern("dd-MM-yyyy HHmm");
+    /* Data Fields */
+    private LocalDateTime recordDate;
```

Comment:
Maybe can consider making these fields `private static final` to make each `Record` object immutable, similar to what they did for `Person` class (ie. To edit a record, need to replace with a new record object)

Reply:
I agree but its should be public final instead